### ISSUE TITLE
pretrained model with instance feature

### ISSUE THREAD

**Issue by LiuNull** [NVIDIA/pix2pixHD#84]
is there any pretrained model with instance features ？
I have trained the model with instance features myself, but It performs well in train set but not good at test set, I wonder is there something wrong with the feature_cluster code?

**Comment by mmreza79** [comment]
Hello @LiuNull, There is no pre-trained model for instance features. Can you share your train and test commend?
